sections/13-acknowledgements.tex

```diff
@@ -0,0 +1,14 @@
+\acknowledgments
+
+We would like to acknowledge financial contributions by Google as part of the Google Summer of Code program.
+This work has made use of data from the European Space Agency (ESA) mission {\it Gaia} (\url{https://www.cosmos.esa.int/gaia}), processed by the {\it Gaia} Data Processing and Analysis Consortium (DPAC, \url{https://www.cosmos.esa.int/web/gaia/dpac/consortium}).
```

Comment:
weird to acknowledge gaia so prominently just as it was used for one of the plot. Maybe a restructuring and rephrasing ("this paper has made use of data")?

Reply:
Yeah this was a merge of the older acknowledgments and the separate file you created.  I wasn't sure what to do with the old one.

Reply:
oh, gotcha. I didn't see the old one 😅

Reply:
I didn't until I was searching for sections.